torchtitan/config_manager.py

```diff
@@ -657,6 +657,24 @@ class Experimental:
     """
 
 
+@dataclass
+class Validation:
+    enabled: bool = False
```

Comment:
Do we need to have a parameter (eg, eval_frequence, integer) to control how many training steps we want to run eval?